configuration/configuration.go

```diff
@@ -247,10 +252,13 @@ func (storage Storage) Type() string {
 		case "redirect":
 			// allow configuration of redirect
 		default:
-			return k
+			if storageType != "" {
+				panic("multiple storage drivers specified in configuration")
```

Comment:
It won't panic if the environment variable is consistent with the storage driver in the config file. But if an environment variable specifies a second storage driver in addition to what's in the config file, that will cause a panic.